admin/controllers/tech-support.js

```diff
@@ -552,6 +553,41 @@ const controller = {
     } catch (error) {
       return next(error)
     }
+  },
+
+  getPsReportRun: async function getPsReportRun (req, res, next) {
+    try {
+      res.locals.pageTitle = 'PS Report Run'
+      req.breadcrumbs('PS Report Run')
+      res.render('tech-support/ps-report-run', {
+        breadcrumbs: req.breadcrumbs(),
+        runReport: req.body?.runReport ?? false,
```

Comment:
So this will be either a string (possibly empty) or a boolean. I don't like giving it more than one type.  Is there a reason for it?

Reply:
it's a string value, as its the state of the checkbox HTML element.  I've tried the various combinations and it has to be parsed from its string value to work.  I've updated that line to reflect this